app.json

```diff
@@ -19,6 +19,9 @@
     },
     {
       "url": "https://github.com/heroku/heroku-buildpack-nginx"
+    },
+    {
+      "url": "https://github.com/moneymeets/python-poetry-buildpack"
```

Comment:
I'm pretty certain that this needs to go before the python buildpack:
> The Python Poetry Buildpack prepares the build to be processed by a Python buildpack such as heroku/python by generating requirements.txt and runtime.txt from poetry.lock. With that said, your repo cannot have a requirements.txt, it will be exported from Poetry (for runtime.txt see below).

https://elements.heroku.com/buildpacks/moneymeets/python-poetry-buildpack